### ISSUE TITLE
Better simplex transform

### ISSUE THREAD

**Issue by jsalvatier** [pymc-devs/pymc#747]
The current simplex transform doesn't put the variables in an unbounded space, and NUTS seems to have difficulty with it.

 STAN uses a "centered stick breaking logit transform" (some code [here](https://groups.google.com/forum/#!topic/stan-dev/09GIjIvhtkE)). Perhaps we should do something similar?

**Comment by aflaxman** [comment]
You could also use softmax http://en.wikipedia.org/wiki/Softmax_function

**Comment by jsalvatier** [comment]
Oh that does seem easy, but it doesn't reduce the dimensionality, so it will be partially non-identifiable. Does it seem like that will lead to problems?

**Comment by jsalvatier** [comment]
Added the stick breaking transform.
